packages/devtools-launchpad/src/index.js

```diff
@@ -93,19 +93,7 @@ function getTargetFromQuery() {
   return null;
 }
 
-function bootstrap(React, ReactDOM, App, appActions, appStore) {
-  const connTarget = getTargetFromQuery();
-  if (connTarget) {
```

Comment:
Actually I just double checked and we do just get the tabs and then in bootstrap we call `getTabs` https://github.com/devtools-html/devtools-core/pull/66/files#diff-4070cf78e7d2f907263e33fe7190d02dR127